options/options.go

```diff
@@ -27,10 +28,13 @@ type ProvenanceOpts struct {
 	ExpectedPackageName *string
 
 	ExpectedPackageVersion *string
+
+	ExpectedBuilderPath string
```

Comment:
nit: ExpectedBuilderIDPath

Reply:
I think we can get rid of this variable entirely, given https://github.com/slsa-framework/slsa-verifier/pull/674/files#r1280023372

Reply:
I agree. The variable has been removed entirely since `verifyBuilderIDPath` function within provenance.go only takes in the provenance and the `expectedBuilderIDPath` now.